core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/groupBy.kt

```diff
@@ -18,6 +18,15 @@ import kotlin.reflect.KProperty
 
 // region DataFrame
 
+/**
+ * Column for grouping can be created inplace
```

Comment:
The comment looks like a good candidate for a NOTE: section, but not like a documentation for a function (probably you leave id with TODO: add docs and with NOTE section